scripts/create_newcase

```diff
@@ -170,6 +170,8 @@ def _main_func():
 
         # Write out the case files
         case.flush(flushall=True)
+        if user_mods_dir is not None and user_mods_dir.startswith(".."):
```

Comment:
This is probably not what you want. Just always call os.path.abspath. If it's already an abspath, no hard is done.